templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -195,10 +195,10 @@ Conditions:
 
 Resources:
   #solr 1
-  OpswxEC2Services1:
+  OpswxEC2Services1AmazonLinux2:
```

Comment:
Could we rename as 'Solr1' since we have split most of its functions to other ec2 instances?

etc

Reply:
Also what was the ruleset we found prior:

Correct me if i'm miss remembering.

I think it was, if Hostname changed it would cause it to re-create, if change resource name it would also create new instance. Both of them would create new instance before remove old. But if it was a time system, it won't wait for the new instance to be ready prior to removal.

Reply:
Sure, that works.